src/views/News/components/NewsList/index.jsx

```diff
@@ -66,42 +85,45 @@ const NewsList = ({ news, personalUnapprovedNews, handleNewsItemEdit, handleNews
     };
   });
 
-  return news.length > 0 || personalUnapprovedNews.length > 0 ? (
-    <Card>
-      {width < BREAKPOINT_WIDTH ? singleHeader : fullHeader}
-      <Grid>
-        <List className={styles.news_list} disablePadding>
-          {personalUnapprovedNews.length > 0 &&
-            personalUnapprovedNews.map((posting) => (
-              <NewsItem
-                posting={posting}
-                unapproved
-                size={width < BREAKPOINT_WIDTH ? 'single' : 'full'}
-                handleNewsItemEdit={handleNewsItemEdit}
-                handleNewsItemDelete={handleNewsItemDelete}
-                key={posting.SNID}
-              />
-            ))}
+  return (
+    <Card style={{ marginBottom: '1rem' }}>
+      <Accordion defaultExpanded={defaultExpanded ?? false}>
+        <AccordionSummary
+          style={headerStyle}
+          expandIcon={<ExpandMore style={{ color: 'white' }} />}
+        >
+          <CardHeader title={header} style={{ padding: '0px' }} />
```

Comment:
In general, it is bad practice to use the `style` attribute to add inline styles. Use scss and add classes to style elements instead. Please replace any `style` attributes you added with classes.